Tools/AP_Periph/of.cpp

```diff
@@ -0,0 +1,47 @@
+#include "AP_Periph.h"
+
+#ifdef HAL_PERIPH_ENABLE_OPTICALFLOW
+
+#include <dronecan_msgs.h>
+
+void AP_Periph_FW::of_init()
+{
+    of.init(-1);
+}
+
+void AP_Periph_FW::of_send_can()
+{
+    const float dt = of.integral_dt();
+    const Vector2f bodyRateIntegral = of.bodyRate() * dt;
+    const Vector2f flowRateIntegral = of.flowRate() * dt;
+
+    com_hex_equipment_flow_Measurement pkt {
+        dt,
+        {flowRateIntegral.x, flowRateIntegral.y},
+        {bodyRateIntegral.x, bodyRateIntegral.y},
+        of.quality()
+    };
+
+    uint8_t buffer[COM_HEX_EQUIPMENT_FLOW_MEASUREMENT_MAX_SIZE];
+    const uint16_t total_size = com_hex_equipment_flow_Measurement_encode(&pkt, buffer, !periph.canfdout());
+
+    canard_broadcast(COM_HEX_EQUIPMENT_FLOW_MEASUREMENT_SIGNATURE,
+                    COM_HEX_EQUIPMENT_FLOW_MEASUREMENT_ID,
+                    CANARD_TRANSFER_PRIORITY_LOW,
```

Comment:
I think this will need to be at least priority MEDIUM, or packet loss may kill this